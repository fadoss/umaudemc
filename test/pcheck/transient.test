# Transient probabilities
#+ req: prism

umaudemc pcheck coin head '@transient(0)'
umaudemc pcheck coin head '@transient(4)'
umaudemc pcheck coin head '@transient(5)'

umaudemc pcheck coin head '@transient(3)' '(thead ; ttail) !'
umaudemc pcheck coin head '@transient(4)' '(thead ; ttail) !'
umaudemc pcheck coin head '@transient(5)' '(thead ; ttail) !'

umaudemc pcheck coin head '@transient(2)' 'choice(1 : thead, 3 : ttail) ; choice(1 : ttail, 3 : thead)' --assign strategy
umaudemc pcheck coin head '@transient(3)' 'choice(1 : thead, 3 : ttail) ; choice(1 : ttail, 3 : thead)' --assign strategy
