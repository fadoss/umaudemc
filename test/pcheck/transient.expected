 1.0                  head
 0.0                  tail
 0.5                  head
 0.5                  tail
 0.5                  head
 0.5                  tail
 1.0                  head
 1.0                  tail
 1.0                  head
 0.75                 head
 0.25                 tail
 0.75                 head
 0.25                 tail
